airflow/providers/google/cloud/hooks/vertex_ai/generative_model.py

```diff
@@ -56,9 +56,13 @@ def get_text_embedding_model(self, pretrained_model: str):
         model = TextEmbeddingModel.from_pretrained(pretrained_model)
         return model
 
-    def get_generative_model(self, pretrained_model: str) -> GenerativeModel:
+    def get_generative_model(
```

Comment:
@CYarros10 if you want to update the name for required parameters in this case you need to deprecate the whole method and create a new method with new names for parameters. Could you please make a changes to this method?

Reply:
I've returned the required parameter name to its original state. thanks for the heads up. no deprecation necessary.